src/store/postSlice.js

```diff
@@ -160,6 +211,12 @@ const postSlice = createSlice({
     [deleteComments.fulfilled]: (state, action) => {
       state.posts = action.payload;
     },
+    [createNewPost.fulfilled]: (state, action) => {
+      state.posts = action.payload;
+    },
+    [editComments.fulfilled]: (state, action) => {
+      state.posts = action.payload;
```

Comment:
suggestion -  handle rejected and pending states as well

Reply:
Thank you for detailed review, Kajal!